Dockerfile

```diff
@@ -39,14 +40,25 @@ RUN mkdir /app/steamcmd /app/sonsoftheforest /app/wine \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
+# Create steam user
+RUN useradd --no-create-home --shell /bin/false steam
+
+# Change ownership
+RUN chown -R steam:steam /app
+
+# Change permissions
+RUN chmod -R 750 /app
+
+# Copy main.sh script
+COPY --chown=steam:steam --chmod=550 main.sh /app/
```

Comment:
Would you mind moving the COPY to just above the CMD, to minimize docker image cache misses?

Reply:
I do not mind, I didn't test the build that way, I assume it is not a problem but I fear a COPY error because of the user switch, to be tested.